src/app/routes/getInitialData/index.test.js

```diff
@@ -62,21 +62,23 @@ describe('getInitialData', () => {
     });
   });
 
-  describe('On client', () => {
+  describe('Using www.test data', () => {
+    const BASE_PATH = 'https://www.test.bbc.com';
     beforeEach(() => {
-      process.env = {};
+      process.env.NODE = true;
+      process.env.RAZZLE_BASE_PATH = BASE_PATH;
     });
 
     it('should call fetch with a relative URL', () => {
       callGetInitialData();
       expect(fetch.mock.calls[0][0]).toEqual(
-        `/data/${defaultServiceParam}/${defaultIdParam}.json`,
+        `${BASE_PATH}/${defaultServiceParam}/articles/${defaultIdParam}.json`,
       );
     });
   });
 
-  describe('On Server', () => {
-    const BASE_PATH = 'https://test.com';
+  describe('Using local data', () => {
```

Comment:
RE: comment above, this isn't actually using local fixture data and the response from `fetch` is mocked, so therefore doesn't need to change.